doc/source/dev/triage.rst

```diff
@@ -0,0 +1,79 @@
+.. _triaging:
+
+============================
+Triaging and curating issues
+============================
+
+SciPy has many hundreds of open issues. Closing invalid ones and correctly
+labelling valid ones (ideally with some first thoughts in a comment) allows
+prioritizing maintenance work and finding related issues easily when working on
+an existing function or subpackage.
+
+While anyone can comment and give more information on open issues, extra
+permissions are needed if you want to apply labels to issues in the SciPy
+repository. While there is no formal process to receive triage rights, the
+expectation is that someone should be active as a contributor before joining
+the team.
+
+Roles and permissions
+=====================
+
+SciPy uses two levels of permissions: triage and core team members. **Triage
+members** can label and close issues and pull requests, while **core team
+members** can label and close issues and pull request, and can also merge pull
```

Comment:
We prefer maintainers over core members now. See also our denomination of teams on GH. https://github.com/orgs/scipy/teams